### PR TITLE
a proposed fix for the undefined cache problem

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: report-processor/build.gradle [hunk 1]
```diff
@@ -30,13 +30,14 @@ dependencies {
     implementation 'org.springframework.retry:spring-retry'
 
     testImplementation project(path: ':rdw-reporting-common', configuration: 'tests')
-    testImplementation 'org.assertj:assertj-core'
```
Comment (malaffoon) [inline report-processor/build.gradle:33]: duplicate

File: report-processor/build.gradle [hunk 2]
```diff
@@ -30,13 +30,14 @@ dependencies {
...
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
         exclude(module: 'commons-logging')
     }
     testImplementation 'org.springframework.cloud:spring-cloud-stream-test-support'
     testImplementation 'org.springframework.security:spring-security-test'
+    testImplementation 'com.fasterxml.jackson.core:jackson-databind'
```
Comment (malaffoon) [inline report-processor/build.gradle:38]: missing, not sure how it was running the test

File: report-processor/src/main/resources/application.yml [hunk 3]
```diff
@@ -23,7 +23,7 @@ spring:
       enabled: false
   cache:
     type: SIMPLE
-    cache-names: translations, subjects, subjectdefinitions
+    cache-names: translations, subjects, subjectdefinitions, jdbcTranslationProvider
```
Comment (malaffoon) [inline report-processor/src/main/resources/application.yml:26]: this is the fix

Comment (gregcharles) [reply report-processor/src/main/resources/application.yml:26]: I mean, it seems right, but I'm not sure how it was working before. I'll build it and give it a try though.

Comment (gregcharles) [reply report-processor/src/main/resources/application.yml:26]: lgtm

**Pull request merged**